functests/performance/updating_profile.go

```diff
@@ -133,6 +133,29 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 				}
 			}
 		})
+
+		It("[test_id:22764] verify that realTimeKernerl is default when realTimeKernel field does not exist", func() {
+			timeout := time.Duration(len(workerRTNodes) * 10)
```

Comment:
hm, your timeout is 10 minutes. If I understand correctly, your goal - check that MCP **does not start updating** after modifying profile, but in your case you are waiting for 10 minutes to get right status (waitForMcpCondition() is "eventually", so it will show you error only if timeout has gone. 10 minutes can be enough for mcp to start update and complete it).

I see 2 options:
Either you reducing timeout (something very small that will not be enough for MCP (1 minute?)) 
Or, you should get MCP status directly from test - without using waitForMcpCondition() function

Reply:
no, `waitForMcpCondition()` returns immediately without error, because the `updating` condition is `false` as expected. The timeout isn't relevant (as long as the test passes as expected). But by doing this in a loop we try to make sure that the update doesn't start within 5 iterations * 5 sec sleep = 25 seconds. That's what I understand at least.

However, as mentioned in my earlier review, there is a func for such an expectation: https://onsi.github.io/gomega/#consistently. Works similar to `Eventually`, but the other way around: instead of waiting for a change until timeout, make sure something doesn't change for some time. I never used that before, but the description sounds good for this usecase.

Reply:
yes, Shanks want to be sure that after modifying profile update was not started in 25 seconds. And if it is so - waitForMcpCondition() returns immediately. But what if updating really started? waitForMcpCondition() will be waiting inside Eventually loop until timeout has reached (10 minutes)

Reply:
The only case where the updating would start is when realTimeKernel is changed to 'true' in any of the previous steps before this test case which is unlikely since the test itself is for updating profile. 

I can put another if condition at the start of the test `if profile.Spec.RealTimeKernel == true` and set it to `False` before running the test, but I think that would be an overkill here. WDYT?

Reply:
> But what if updating really started? 

ah right, we might not catch the negative case, when the update starts but finishes within 10 minutes. But this will also be fixed by using `Consistently` instead of loop + `Eventually` (in the wait func)

> I can put another if condition at the start of the test if profile.Spec.RealTimeKernel == true and set it to False

What about failing the test if it's true. Less overkill, but would ensure that we catch if setting changes in test setup / earlier tests.

Reply:
> > I can put another if condition at the start of the test if profile.Spec.RealTimeKernel == true and set it to False
> 
> What about failing the test if it's true. Less overkill, but would ensure that we catch if setting changes in test setup / earlier tests.

On second thought, don't think it would be overkill since it would be skipped anyway. :-)

Reply:
I would change _if profile.Spec.RealTimeKernel == true_ to  _Expect(profile.Spec.RealTimeKernel).To(smth)_ 
IMHO, in tests Expect is more readable then If